cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -162,6 +239,56 @@ def listen_for_course_catalog_info_changed(sender, signal, **kwargs):
     )
 
 
+@receiver(COURSE_CERTIFICATE_CONFIG_CHANGED)
+def listen_for_course_certificate_config_changed(sender, signal, **kwargs):
+    """
+    Send course certificate config data onto the Credentials service to create or change one.
+    """
+    certificate_config = kwargs['certificate_config']
+    files_to_upload = dict(get_certificate_signature_assets(certificate_config))
+    try:
+        credentials_client = get_credentials_api_client(
+            User.objects.get(username=settings.CREDENTIALS_SERVICE_USERNAME),
+        )
+        credentials_api_base_url = get_credentials_api_base_url()
+        api_url = urljoin(f'{credentials_api_base_url}/', 'course_certificates/')
+        payload = attr.asdict(certificate_config)
+        payload['course_key'] = str(payload.pop('course_key'))
+        response = credentials_client.post(
+            api_url,
+            files=files_to_upload,
+            json=payload
+        )
+        response.raise_for_status()
+        log.info(f'Course certificate config sent for course {certificate_config.course_key} to Credentials.')
+    except Exception:  # lint-amnesty, pylint: disable=W0703
+        log.exception(f'Failed to send course certificate config for course {certificate_config.course_key}.')
+
+
+@receiver(COURSE_CERTIFICATE_CONFIG_DELETED)
+def listen_for_course_certificate_config_deleted(sender, signal, **kwargs):
+    """
+    Send course certificate config data onto the Credentials service to delete one.
+    """
+    certificate_config = kwargs['certificate_config']
+    try:
+        credentials_client = get_credentials_api_client(
+            User.objects.get(username=settings.CREDENTIALS_SERVICE_USERNAME),
+        )
+        credentials_api_base_url = get_credentials_api_base_url()
+        api_url = urljoin(f'{credentials_api_base_url}/', 'course_certificates/')
+        payload = attr.asdict(certificate_config)
+        payload['course_key'] = str(payload.pop('course_key'))
```

Comment:
And here too.

Reply:
Yep, fixed here https://github.com/raccoongang/edx-platform/pull/2465/files.